app/helpers/admissions_helper.rb

```diff
@@ -35,4 +35,12 @@ def facet_count_for_gender(value)
   def default_value(value)
     params[value]
   end
+
+  def friendly_date(value)
```

Comment:
I'd have to check to see what's faster, but another way to do this in fewer lines is with:

```ruby
year, month, day = value.to_s.scan(%r{ (\d{4})(\d{2})(\d{2}) }.capture
"#{month}/#{day}/#{year}"
```